LICENSE.txt

```diff
@@ -222,7 +222,7 @@ Additional Notices: N.a.
 Component: Dependency Finder
 Licensor: Jean Tessier
 Website: https://github.com/jeantessier/dependency-finder
-License: Custom license (https://github.com/jeantessier/dependency-finder/blob/master/license.txt, a copy is included below)
+License: BSD-like (https://github.com/jeantessier/dependency-finder/blob/master/license.txt, a copy is included below)
```

Comment:
This is a literal copy of [1] (the only difference is that the maintainer replaced "copyright holder" by his name, which is  ok since he is the only contributor to this project) hence I think this should be named "3-Clause BSD License". I think rather the OSI is the authority about OSS licensing than a single maintainer of an open source project.

[1] https://opensource.org/licenses/BSD-3-Clause

Reply:
There is also another small change.

```diff
-IN NO EVENT SHALL THE REGENTS OR CONTRIBUTORS
+IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS
```

I'm no expert in licensing, so I will ping @henrikplate and @copernico which may have more experience than me.

Reply:
@msohn is right, from a practical standpoint there is probably not much difference from the 3-clause BSD. However, since I do not know what are the exact legal implications of replacing *COPYRIGHT HOLDER* with *REGENTS* and also since Jean Tessier himself calls the license **BSD-like** in the project website, I would stick to that terminology.

Reply:
"BSD-like" is too vague in my opinion - people will need to dig deeper anyhow. I will check with the license team whether they agree to call it a "3-Clause BSD License" (regardless of the change of words), otherwise I would keep "Custom License".

Reply:
The license team supports the use of "BSD-like license", thus, I will approve the PR. Thank you @msohn for pointing us to this issue.